src/components/EntityThumbnailUploader/ThumbnailUploaderProvider.tsx

```diff
@@ -0,0 +1,52 @@
+import { createContext, useState } from "react"
+
+import useCreateContext from "@hooks/useCreateContext"
+import { $Any } from "@types"
+
+const ThumbnailUploadContext = createContext<{
+  onContextMenu?: Function
+  resetFileUploadState?: Function
+  inputRef?: HTMLInputElement
+}>({})
+
+type Props = {
+  handleThumbnailUpload: (thumbnails: $Any[]) => {}
+  entities: $Any
+  inputRef: $Any;
+  children?: JSX.Element|JSX.Element[];
+}
+
+const ThumbnailUploadProvider = ({
+  children = [],
+  inputRef,
+}: Props) => {
+
+  const [_, setFileUploadInProgress] = useState(false)
+  const [ctxMenuShow] = useCreateContext()
+  const resetFileUploadState = () => setFileUploadInProgress(false)
+
+  const ctxMenuItems = () => [
+    {
+      label: 'Upload New Thumbnail',
```

Comment:
Only capilized first letter


```suggestion
      label: 'Upload new thumbnail',
```

Reply:
And also use the `add_photo_alternative` icon to match the dropzone icon.